Scripts/RemediationScripts/AutoRemediation.ps1

```diff
@@ -162,7 +162,8 @@ Enter the choice (1|2)";
                 ($control.ControlId -eq "Azure_DBForMySQLFlexibleServer_DP_Use_Secure_TLS_Version_Trial") -or
                 ($control.ControlId -eq "Azure_SQLDatabase_DP_Enable_TDE") -or
                 ($control.ControlId -eq "Azure_FrontDoor_DP_Use_Secure_TLS_Version_Trial")-or
-                ($control.ControlId -eq "Azure_FrontDoor_CDNProfile_DP_Use_Secure_TLS_Version_Trial")){
+                ($control.ControlId -eq "Azure_FrontDoor_CDNProfile_DP_Use_Secure_TLS_Version_Trial") -or
+                ($control.ControlId -eq "Azure_SQLServer_AuthN_Dont_Allow_Public_Network_Access")){
```

Comment:
also changes in file Scripts/RemediationScripts/ControlsEligibleForRemediationThroughUI.md to list control in remediation mode via UI

Reply:
Added.